.devcontainer/rclone/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+echo "Downloading rclone sources from ${RCLONE_REPOSITORY}:${RCLONE_REF}"
+mkdir -p /tmp/rclone
+cd /tmp/rclone
+git clone "${RCLONE_REPOSITORY}"
+cd rclone
+git checkout "${RCLONE_REF}"
+
+echo "Building rclone"
+make rclone
+cd $HOME
+rm -rf /tmp/rclone
```

Comment:
suggestion: It will be much faster if you dont to build rclone from source. Rclone publishes executables in their github releases. https://github.com/rclone/rclone/releases. So you can just download and extract that.

Reply:
The build step is not too long, at least much faster than the SOLR step anyway. This is also very simple to do to test fixes and changes on `rclone`, which we will soon have (see #741).